client/state/comments/actions.js

```diff
@@ -20,6 +20,8 @@ import {
 } from '../action-types';
 import { NUMBER_OF_COMMENTS_PER_FETCH } from './constants';
 
+export const requestComment = ( { siteId, commentId } ) => ( { siteId, commentId } );
```

Comment:
also, i'm surprised you're not using FSA style actions? just trying to keep with the other comment action layout?